src/components/Reports/LocationSection.tsx

```diff
@@ -437,3 +453,27 @@ function InfoField(props: InfoFieldProps): JSX.Element {
     />
   );
 }
+
+export const buildStartedDateValid = (loc: ReportSeedBank | ReportNursery) => {
+  if (loc.buildStartedDate && loc.buildCompletedDate) {
+    return Date.parse(loc.buildStartedDate) <= Date.parse(loc.buildCompletedDate);
+  }
+  return true;
+};
+
+export const buildCompletedDateValid = (loc: ReportSeedBank | ReportNursery) => {
+  if (loc.buildStartedDate && loc.buildCompletedDate) {
+    return Date.parse(loc.buildStartedDate) <= Date.parse(loc.buildCompletedDate);
+  }
+  if (loc.buildCompletedDate && loc.operationStartedDate) {
+    return Date.parse(loc.buildCompletedDate) <= Date.parse(loc.operationStartedDate);
+  }
+  return true;
+};
```

Comment:
do we want both checks to be executed and not short-circuit on the first one?

Reply:
hm, good point, if the first one resolves to `true` and the second one would have resolved to `false` then it doesn't get to return that. I'll think about how to restructure this logic for that.

Reply:
The changes look good 👍